Library/ViewModels/RewardCellViewModel.swift

```diff
@@ -304,10 +303,23 @@ private func minimumRewardAmountTextColor(project: Project, reward: Reward) -> U
   }
 }
 
+private func needsConversion(project: Project) -> Bool {
+  guard let currentCurrency = project.stats.currentCurrency else {
+    return needsConversion(projectCountry: project.country,
+                           userCountry: AppEnvironment.current.config?.countryCode)
+  }
+  return currentCurrency == AppEnvironment.current.apiService.currency
```

Comment:
If our API ever returns a different currency than the one we send along, we fall back to USD here. We may wanna reconsider that logic, though?

Reply:
Could we not keep these defaults on the `Config` and lens them in the first time?

Reply:
Ah, I mistyped. I mean we don't show any conversion info.